Assets/[SCRIPTS]/Movement.cs

```diff
@@ -8,7 +8,7 @@ public class Movement : MonoBehaviour
     // Start is called before the first frame update
     void Start()
     {
-        speed = 0.1f;
+        speed = 10f;
```

Comment:
If is public initialize from inspector not here, remove line, also access modifier for this should be [SerializeField] private